### PR TITLE
Standardize format for commands

### LINKED ISSUE #316: Standardize command format in UG

### LINKED ISSUE #317: Create command information

### LINKED ISSUE #318: Update command summary

### LINKED ISSUE #319: Add warning for invalid prefixes

### PR DESCRIPTION
**Pull request by AudreyFelicio**

### CODE REVIEW

**Review by Ashley-Lau — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -89,30 +89,67 @@ This section provides support for the commands required to perform desired actio
...
+  but `update sn/Fairprice1 n/Apple n/Banana` is not a valid command format.
+
+* Compulsory prefixes with `...` trailing after them can be used more than one time in one command. <br>
+  For example, in `delete sn/<serial number>...`, the `sn/<serial number>` must be provided and duplicates are allowed. <br>
+  `delete sn/Fairprice1` and `delete sn/Fairprice1 sn/Ntuc1` are both valid command formats.
   
+* Multiple prefixes combined with `|` means only one of them can be provided at a single command. <br>
+  For example, in `update sn/<serial number> [iq/<increment value> | nq/<new quantity>]`, at most one of `iq/<increment value>`
+  or `nq/<new quantity>` may be provided. <br>
+  `update sn/Fairprice iq/+10 nq/100` is an invalid command format, but `update sn/Fairprice1 iq/+10`,
```
Comment (Ashley-Lau) [inline docs/UserGuide.md]: Is there supposed to be a 1 behind the fairprice here?

**Review by Ashley-Lau — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (AudreyFelicio) [reply docs/UserGuide.md]: Yes. Thanks for mentioning!

**Pull request merged**